emirge.py

```diff
@@ -877,7 +873,7 @@ def cluster_sequences2(self, fastafilename):
         already_removed = set()  # seq_ids
         # this is a bit slow and almost certainly could be sped up with algorithmic improvements.
         times = []  # DEBUG
-        for row in csv.reader(file("%s.us.txt"%tmp_fastafilename), delimiter='\t'):
+        for row in csv.reader(open("%s.us.txt"%tmp_fastafilename), delimiter='\t'):
```

Comment:
wrap in `with`

Reply:
As above, I would politely suggest this be done as a separate issue / pull request.